sql/moz-fx-data-shared-prod/telemetry_derived/attributed_windows_desktop_funnel_sessions_v1/query.sql

```diff
@@ -0,0 +1,60 @@
+-- Query for telemetry_derived.attributed_windows_desktop_funnel_sessions_v1
+CREATE TEMP FUNCTION map_country_name_to_prefix(input STRING)
+RETURNS STRING
+LANGUAGE js
+AS
+  """
+    if (input === "USA") {
+        return "US";
+    } else if (input === "Germany") {
+        return "DE";
+    } else if (input === "United Kingdom") {
+        return "GB";
+    } else if (input === "France") {
+        return "FR"
+    } else if (input === "Canada") {
+        return "CA"
+    } else if (input === "Brazil") {
+        return "BR"
+    } else if (input === "Mexico") {
+        return "MX"
+    }
+    return "OTHER";
+    """;
+
+CREATE TEMP FUNCTION bucket_country(country STRING) AS (
+  CASE
+  WHEN
+    country IN ('US', 'CA', 'DE', 'FR', 'GB')
+  THEN
+    [country, 'tier-1']
+  WHEN
+    country IN ('MX', 'BR')
+  THEN
+    [country, 'non-tier-1']
+  ELSE
+    ['non-tier-1']
+  END
+);
+
```

Comment:
@fbertsch I'm with @SuYoungHong -- I'd prefer to avoid expressing "tier 1" vs "rest of world" in this dataset, and just express that somehow with Looker (or the query accessing this dataset, in the cases where we fall back to redash). This is the sort of thing we could express with lookml, right? Am I safe to just leave it out of here for now?